app/helpers/dashboard_helper.rb

```diff
@@ -8,4 +8,31 @@ def solrstatus_to_bootstrap_item(status)
              end
     "list-group-item-#{suffix}"
   end
+
```

Comment:
Some mixing of tab and space indentation in this file
<img width="611" alt="Screen Shot 2021-04-22 at 2 46 41 PM" src="https://user-images.githubusercontent.com/458247/115769533-a46e6280-a379-11eb-922f-c7d5504c4d16.png">